### DISCUSSION TITLE
How to get User Fields from search_recent_tweets query="conversation_id:..."

### DISCUSSION THREAD

**Discussion by RobinManoli** [tweepy/tweepy#2213] [Questions]
### Question

My query is working when I don't add any user fields or tweet fields, but as soon as I add them I get the errors:
`Duplicate parameters are not allowed: the `user.fields` query parameter`
or
`Duplicate parameters are not allowed: the `tweet.fields` query parameter`

### Relevant Code

```python
user_fields = (
        [
            "name",
            "username",
            "description",
            "location",
        ],
    )

    tweet_fields = (
        [
            "author_id",
            # "conversation_id",
            "id",
            "public_metrics",
            "referenced_tweets",
            "text",
        ],
    )

    tweets = client.search_recent_tweets(
        query="conversation_id:...",
        tweet_fields=tweet_fields,
        expansions="author_id",
        user_fields=user_fields,
    )
```

### Traceback

```python-traceback
tweepy.errors.BadRequest: 400 Bad Request
Duplicate parameters are not allowed: the `tweet.fields` query parameter
Duplicate parameters are not allowed: the `user.fields` query parameter
```

### Twitter API Access Plan

Free

### Tweepy Version

4.14.0

### Checklist

- [X] I have searched for duplicate questions.
- [X] If applicable, I have removed any visible credentials from my code and/or screenshots.

**Comment by deelv25** [comment]
Hey, I recently had this same problem, using python requests library but same error. The fix ended up being structuring the 'tweet_fields' and 'user_fields' params as single strings rather than arrays. Here's what worked for me:
```
params = {
    'tweet.fields': 'created_at,attachments',
    'user.fields': 'name,username,id'
}
response = requests.get(ROOT_TWEETS_URL+post_id, params=params, headers=headers)
```
